src/firmware/app/world/firmware_robot.c

```diff
@@ -52,6 +54,8 @@ FirmwareRobot_t* app_firmware_robot_create(
     new_robot->robot_constants            = robot_constants;
     new_robot->controller_state           = controller_state;
 
+    TLOG_INFO("new robot has been created!");
```

Comment:
this is for demo purposes, this log will show up on sim, will be removed before merging this pr